Samples~/LWRP/.sample.json

```diff
@@ -0,0 +1,5 @@
+{
+	"displayName":"LWRP Support",
```

Comment:
If I remember right, the upm-ci will add entries for the samples in package.json during the build process. I don't know if that's still valid.
Otherwise, we would need to add them manually.

Reply:
I tried running it through `upm-ci`, and at least locally it does not add the samples to the manifest. Added in 28178654